src/ConversionContext.h

```diff
@@ -16,7 +16,16 @@ namespace drafter {
         refract::Registry registry;
 
     public:
+        bool generateSourceMap;
```

Comment:
uh, when I argue by "future" it is wrong and have to remove code :)

but there is another argue - pollution of class interface with every one new member.
It should be as clear as possible.

Related things should grouped.